.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+**Description:**  
+New tool, Bug fixing, or Improvement?  
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.  
+For new tools, provide total size and installation time.
```

Comment:
LGTM!

Could you please change line 4 like this (make it bold and add "please" word 🙂 ):
**For new tools, please provide total size and installation time.**

Reply:
Corrected wording.
I didn't add bold because it looks a bit strange for me (bold is used for titles in this file). Please let me know if you insist

Reply:
I'd like customers to pay attention that we need tool size and installation time because most likely we'll ask them to provide this info in case of absence, does it make sense?

Reply:
Updated. Please revisit